src/tests/__snapshots__/cssNamespace.test.js.snap

```diff
@@ -19,8 +19,10 @@ export default MyStyledComponent;
 
       ↓ ↓ ↓ ↓ ↓ ↓
 
-const styled = { div() {} };
```

Comment:
Some slight formatting changes to the output after updating some of the dependencies.